gips/datahandler/worker.py

```diff
@@ -55,6 +55,14 @@ def fetch(driver, asset_type, tile, date):
     DataClass  = utils.import_data_class(driver)
     AssetClass = DataClass.Asset
     filenames  = AssetClass.fetch(asset_type, tile, date)
+    if len(filenames) == 0:
+        asset = dbinv.models.Asset.objects.get(
+            driver=driver, asset=asset_type, tile=tile, date=date)
+        asset.status = dbinv.models.Status.objects.get(status='failed')
```

Comment:
I don't think we have a status table any more; just save it like it was done on line 51 above.  Also no need to `get()` the `asset` again as it's already present in this scope.  Otherwise good idea to save a failed state!